### DISCUSSION TITLE
Configuration is not changing with a new configuration file on Windows

### DISCUSSION THREAD

**Discussion by LWmp1** [SafeExamBrowser/seb-win-refactoring#136] [Q&A]
Hello

We have BYOD at our school and use the SEB on Windows and macOS.
Because of a safety issue we wanted to update the SEB on every device and configurate it new.
 
Therefor we created 2 new configurations files, with new settings and passwords. 
Our students deinstalled the SEB and downloaded the newes version.
We gaved them the new configuration file and the teachers entered the new password for the new configuration and had to confirm the change with the old password.

The process  worked with all Macs and most of the Windows devices.

But on some Windows devices the configuration did not change and the old password of the previous configuration was still active. 
We already tried to install an older Version or delete the configurationsettings in the folder "ProgramData" but it did not worked.
Is there another way how we can aktivated the new configuration on the devices of our students or get rid of the old one?

Thanks for the help

**Comment by dbuechel** [comment]
SEB configuration files which are intended to be used as local client configuration may be placed in two directories on a user's computer:

`%ProgramData%\SafeExamBrowser\SebClientSettings.seb`
`%AppData%\SafeExamBrowser\SebClientSettings.seb`

So to resolve your issue, instruct the students to make sure that both of the above mentioned client configurations are deleted. See also https://safeexambrowser.org/windows/win_usermanual_en.html#ConfigFilePane.
